recipes/py2neo/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name='py2neo' %} 
+{% set version='4.0.0b2' %}
+{% set md5='d7ff00b7af97241f104be88b30cd2e45' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/97/3c/6257962dd23ad791d6e885116b734e55602980c862b028e684b38cc1d96c/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  patches:
+    - console.patch
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - py2neo = py2neo.cli.tool:main
+    - neokit = neokit:main
+    - geoff = py2neo.ext.geoff.__main__:main
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - neo4j-python-driver ==1.1.0rc1
```

Comment:
It is specified like that (the only requirement with a specific version specified) in py2neo's 4.0.0b2's setup.py.